subsys/bluetooth/host/direction.c

```diff
@@ -351,6 +351,7 @@ void hci_df_prepare_connectionless_iq_report(struct net_buf *buf,
 	report->cte_type = BIT(evt->cte_type);
 	report->packet_status = evt->packet_status;
 	report->slot_durations = evt->slot_durations;
+	report->per_evt_counter = evt->per_evt_counter;
```

Comment:
```suggestion
	report->per_evt_counter = sys_le16_to_cpu(evt->per_evt_counter);
```

This may also be missing for other fields

Reply:
Thanks, added this.

Reply:
Good catch @Thalley thank you!

Reply:
@ppryga @jakkra Looks like the RSSI is missing a `sys_le16_to_cpu`, will one of you add a PR to fix that one, or add it to this PR?

Reply:
Handled for RSSI in a new commit.